### ISSUE TITLE
Add support for geoLocationPolygon entries as part of the json API attributes

### ISSUE THREAD

**Issue by richardhallett** [datacite/lupo#327]
Add support for geoLocationPolygon entries as part of the json API attributes

This should allow you to send a full geoLocation entry with associated geoLocationPolygon elements as you would in the XML.

**Comment by kjgarza** [comment]
more detail in to fix: https://app.frontapp.com/open/msg_4ieecub

**Comment by richardhallett** [comment]
Still a requirement, causes some confusion in regards to what is and isn't allowed via the attributes.

**Comment by KellyStathis** [comment]
Hi @nensor - it is not currently possible to provide geoLocationPolygon attributes as JSON via the REST API. However, this metadata can be provided via XML as described here: https://support.datacite.org/docs/api-create-dois#provide-metadata-in-formats-other-than-json/

I took a look at your example and it looks like the geoLocationPolygon field is there: https://api.test.datacite.org/dois/10.80447/23903. Was this created by submitting metadata in XML format?

**Comment by nensor** [comment]
Yes, I've gone back to submitting in XML format for this information.  Upcoming projects were transitioning to JSON, but running across this issue has forced a regression in this case.

**Comment by nensor** [comment]
I should note, the program in question is still using DataCite's old (?) MDS service exclusively in XML, which was what I was intending to transition from.  Is the only really viable way to submit this is essentially creating an entire XML string in memory, encode that to base64, and tack that in via an "xml" attribute in order to support this functionality?  That seems rather excessive.

**Comment by tmorrell** [comment]
This is still an issue
